TipCatDotNet.Api/Services/Payments/TransactionService.cs

```diff
@@ -94,6 +101,32 @@ async Task<Result> SetPaymentTime(int memberId)
     }
 
 
+    public async Task<Result<List<FacilityTransactionResponse>>> Get(MemberContext context,
+        TransactionFilterProperty filterProperty, CancellationToken cancellationToken = default)
+    {
+        var facilities = await _context.Facilities
+            .Where(f => f.AccountId == context.AccountId)
+            .ToListAsync(cancellationToken);
+
+        var transactions = _context.Transactions
+            .Select(TransactionProjection())
+            .AsEnumerable()
```

Comment:
Do we need `AsEnumerable` here?

Reply:
Also, why it's sync?

Reply:
I tried Async but AsAsyncEnumerable returns IAsyncEnumerable which doesn't contain GroubBy method